src/Features/VisualBasic/Portable/InitializeParameter/VisualBasicAddParameterCheckCodeRefactoringProvider.vb

```diff
@@ -43,5 +43,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InitializeParameter
         Protected Overrides Function GetBody(functionDeclaration As SyntaxNode) As SyntaxNode
             Return InitializeParameterHelpers.GetBody(functionDeclaration)
         End Function
+
+        Protected Overrides Function GetParameterNodeAtIndex(position As Integer, functionDeclaration As SyntaxNode) As SyntaxNode
+            Throw New NotImplementedException()
```

Comment:
Don't forget implement this in VB.

Reply:
I'll open a ticket for that for now. Will try to accomplish soon.